R/msf_dict_rename_helper.R

```diff
@@ -2,38 +2,68 @@
 #'
 #' @export
 #' @param disease Specify which disease you would like to use.
-#'   Currently supports "Cholera", "Measles" and "Meningitis", "AJS".
+#'   Currently supports "Cholera", "Measles", "Meningitis", "AJS",
+#'   "Mortality", "Nutrition", "Vaccination_short" and "Vaccination_long".
 #'
 #' @param varnames Specify name of column that contains varnames. Currently
-#'   default set to "Item".  (this can probably be deleted once dictionaries
-#'   standardise) If `dictionary` is "Mortality", `varnames` needs to be "column_name"`.
-#' 
+#'   default set to "data_element_shortname".
+#'   If `dictionary` is a survey ("Mortality", "Nutrition", "Vaccination_short"
+#'   or "Vaccination_long") `varnames` needs to be "name"`.
+#'
 #' @param copy_to_clipboard if `TRUE` (default), the rename template will be
 #'   copied to the user's clipboard with [clipr::write_clip()]. If `FALSE`, the
 #'   rename template will be printed to the user's console.
 #'
 #' @return a dplyr command used to rename columns in your data frame according
 #' to the dictionary
 msf_dict_rename_helper <- function(disease, varnames = "data_element_shortname", copy_to_clipboard = TRUE) {
-  # get msf disease specific data dictionary
-  dat_dict <- msf_dict(disease = disease, compact = TRUE)
+
+  # make disease name lower case
+  disease <- tolower(disease)
+
+  # get msf disease specific outbreak data dictionary
+  if (disease == "cholera" | disease ==  "measles" |
+      disease == "meningitis" | disease == "ajs") {
+
+    dat_dict <- msf_dict(disease = disease, compact = TRUE)
+
+    # define the name of the column in the dictionary which has variable class
+    var_type_col <- "data_element_valuetype"
+  }
+
+  # get msf disease specific survey data dictionary
+  if (disease == "mortality" | disease == "nutrition" |
+      disease == "vaccination_short" | disease == "vaccination_long") {
+
+    dat_dict <- msf_dict_survey(disease, compact = TRUE)
+
+    # define the name of the column in the dictionary which has variable class
+    var_type_col <- "type"
+  }
+
+  # change vaccinations to be one
+  if (disease == "vaccination_short" | disease == "vaccination_long") {
+
+    disease  <- "vaccination"
+  }
+
   # get the outbreak Rmd to check if the variable is optional or required
   outbreak_file <- available_sitrep_templates(recursive = TRUE, pattern = ".Rmd", full.names = TRUE)
```

Comment:
@zkamvar just a quick one - I added in surveys to the rename helper; but `available_sitrep_templates()` is part of {sitrep}, do you think we should import to {epidict}? 
All of {epidict} is reexported in {sitrep} - just wondering if we think ppl might use epidict in isolation .... but I guess not!